### ISSUE TITLE
Documentation missing for port option for connect flowop

### ISSUE THREAD

**Issue by dustinblack** [uperf/uperf#53]
It appears that the connect flowop supports a port option, but this option does not appear in the documentation.

https://github.com/uperf/uperf/blob/0425fa8be9d706557cc808b2e8377a03ed31059a/src/flowops_library.c#L157

**Comment by realneel** [comment]
yeah. You can use something like this (just add port=XXXX in the profile).
```
<?xml version="1.0"?>
<profile name="iPERF">
  <group nthreads="$nthr">
        <transaction iterations="1">
            <flowop type="connect" options="remotehost=$h protocol=$proto port=8080
	    wndsz=50k  tcp_nodelay"/>
        </transaction>
        <transaction duration="30s">
            <flowop type="write" options="count=10 size=8k"/>
        </transaction>
        <transaction iterations="1">
            <flowop type="disconnect" />
        </transaction>
  </group>

</profile>
```

**Comment by realneel** [comment]
@dustinblack Do you plan to use the `port` option? I can add the use case to the doc.

I was thinking of the following

- `port`: You can specify a port to connect (or accept). This is useful in cases where specific ports can accept connections. Note that this only controls the initial connect or accept. The actual data transfer happens on a different port as accept creates a new socket to handle the connection.

**Comment by dustinblack** [comment]
Hi @realneel -- We use the `port` option in two different benchmark workflow projects already:

https://github.com/arcalot/arcaflow-plugins/blob/78da00b52e7d943148e5aa76349eca8f8a71b8a7/python/uperf/uperf_schema.py#L82-L87

https://github.com/cloud-bulldozer/benchmark-operator/blob/0e6043a1473c895ea381712effa35e4c27808850/roles/uperf/templates/configmap.yml.j2#L26

This is important when running uperf in kubernetes containers with nodeport enabled. See:

https://github.com/cloud-bulldozer/benchmark-operator/issues/757

and

https://github.com/cloud-bulldozer/benchmark-operator/pull/721
